egs2/TEMPLATE/asr1/asr.sh

```diff
@@ -951,12 +951,18 @@ if [ ${stage} -le 5 ] && [ ${stop_stage} -ge 5 ] && ! [[ " ${skip_stages} " =~ [
             log "Error: not supported SOT training for whisper token_list"
             exit 2
         fi
+
+        _opts=""
+        if [ "${token_type}" = "whisper_multilingual" ]; then
```

Comment:
Would default `lang=noinfo` work here?

Reply:
I add a `LANGUAGES_CODE_MAPPING` to map the language codes of ESPnet with the language IDs Whisper and make sure the input language code is supported by the Whiper models.